### ISSUE TITLE
besside-ng not work properly for wpa-handshake capture

### ISSUE THREAD

**Issue by jbenden** [jbenden/aircrack-ng-migration#801]
**Reported by Jano on 31 Oct 2010 21:59 UTC**

Hi to all,

Besside-ng capture and writes the valid handshake without capturing the name of ESSID

```
jano:~$ aircrack-ng -w dict-test.txt wpa.cap 
Opening wpa.cap
Read 3 packets.

   #  BSSID              ESSID                     Encryption

   1  00:24:B2:xx:xx:xx                            WPA (1 handshake)

Choosing first network as target.

Opening wpa.cap
An ESSID is required. Try option -e.
```

```
# SSID              | KEY                                    | BSSID
                    | Got WPA handshake                      | 00:24:b2:xx:xx:xx
```

Bye Jano

Migrated-From: https://trac.aircrack-ng.org/ticket/801

**Comment by jbenden** [comment]
**Comment by sorbo on 31 Oct 2010 21:59 UTC**

Try revision 1796.  I added hidden SSID support.  Let me know if it works.

**Comment by jbenden** [comment]
**Comment by Jano on 31 Oct 2010 21:59 UTC**

Sorbo,

- Now also works with hidden ESSID, and save the 4-way-handshake correctly.

Jano

**Comment by jbenden** [comment]
**Comment by misterx on 31 Oct 2010 21:59 UTC**

Fixed in aircrack-ng/aircrack-ng@361d1be54dc1cd226489474af0e06ae006f82e98.
